winternlc/config.py

```diff
@@ -39,9 +41,10 @@ def get_correction_dir(version: str = LATEST_ZENODO_VERSION) -> Path:
     :return: Path to the correction data directory
     """
 
+    available_versions = ["v0.1", "v1.0", "v1.1", "v2.0"]
     assert (
-        version in ZENODO_URL_MAP
-    ), f"Version {version} is not available. Please choose from {ZENODO_URL_MAP.keys()}"
+        version in available_versions
```

Comment:
What was the reason for this change? AFAICT the keys of `ZENODO_URL_MAP` should be these same four values, so this is just an extra duplication of the same info.